test/index.test.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable */
+import validateOptions from '../dist/validateOptions';
```

Comment:
You keep defaulting to this testing approach and as I have said before, you negate most of the usefulness of using Jest.

- Jest doesn't require a build step as it's setup in defaults. babel-jest takes care of that.
- Jest runs tests async & only runs the tests related to the code you have changed, this negates that as well.
- None of the editor plugins for Jest work with this approach